cinema/serializers.py

```diff
@@ -0,0 +1,23 @@
+from rest_framework import serializers
+from cinema.models import Movie
+
+
+class MovieSerializer(serializers.ModelSerializer):
+    id = serializers.IntegerField(read_only=True)
+    title = serializers.CharField(required=True)
```

Comment:
The `title` field is marked as required, but there is no validation logic to handle potential validation errors. Ensure you handle validation errors properly, possibly by using `raise_exception=True` when calling `serializer.is_valid()` in your views.